R/statuses.R

```diff
@@ -154,11 +154,11 @@ setMethod("show", signature="status", function(object) {
 })
 
 updateStatus <- function(text, lat=NULL, long=NULL, placeID=NULL,
-                         displayCoords=NULL, inReplyTo=NULL, mediaPath=NULL, ...) {
+                         displayCoords=NULL, inReplyTo=NULL, mediaPath=NULL, bypassCharLimit=F, ...) {
```

Comment:
Can you change F to FALSE? The latter has been preferred for as long as I can remember